src/App.js

```diff
@@ -1,9 +1,35 @@
+import { Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Pages/Home';
+import Quote from './components/Pages/Quote';
 
-const App = () => (
-  <div className="container">
-    <Calculator />
-  </div>
-);
+function App() {
+  return (
+    <Routes>
+      <Route
+        path="/"
+        element={(
+          <Calculator />
+        )}
+        exact
+      />
+      <Route
+        path="/Home"
+        element={
+          <Home />
+        }
+        exact
+      />
```

Comment:
- [ ] Let's make the `<Home />` to have the root path route `/` and `<Calculator />` to have `/Calculator` path as the home page is always the root path. Also, it's a requirement. See suggestion:

```suggestion
    <Routes>
      <Route
        path="/"
        element={(
           <Home />
        )}
        exact
      />
      <Route
        path="/Calculator"
        element={
          <Calculator />
        }
        exact
      />
```